src/dac.rs

```diff
@@ -0,0 +1,61 @@
+use riot_sys::dac_t;
+
+#[derive(Debug)]
+pub struct DACLine {
+    line: dac_t,
+}
+
+#[derive(Debug)]
+#[non_exhaustive]
+pub enum DACError {
+    /// The given dac_t line did not exist
+    NoLine,
+    /// An unknown error did occur
+    Unknown,
+}
+
+impl DACLine {
+    /// Creates and intializes a new [`DACLine`] from the given
+    /// unitialized [`dac_t`].
+    ///
+    /// Returns an Error if the given line does not exist
+    /// on the board.
+    pub fn new(line: dac_t) -> Result<Self, DACError> {
```

Comment:
Would you mind trying this out with https://github.com/RIOT-OS/rust-riot-wrappers/issues/37#issuecomment-1430112835 ?

Reply:
It seems that [`DAC_LINE`](https://doc.riot-os.org/group__drivers__periph__dac.html) is missing from `riot_sys/build.rs` which seems to be the equivalent for `*_DEV` in the DAC interface.

Reply:
Sorry, I missed that reply. A version of -sys that adds DAC_LINE is [being merged](https://github.com/RIOT-OS/rust-riot-sys/pull/22), and should thus be usable easily in a few mintues

Reply:
Thank you, is now done in https://github.com/RIOT-OS/rust-riot-wrappers/pull/36/commits/25dead1549558fe43da4a4b02958c2a5d9df45a2